src/test/suite/views/webview-app/app.test.tsx

```diff
@@ -0,0 +1,21 @@
+import assert from 'assert';
+import * as React from 'react';
+import sinon from 'sinon';
+import * as featureFlags from '../../../../featureFlags';
+import { render, screen } from '@testing-library/react';
+import App from '../../../../views/webview-app/app';
+
+describe('App Component Test Suite', () => {
+  afterEach(() => sinon.restore());
+  test('it renders the old overview page when useNewConnectionForm is falsy', () => {
+    sinon.stub(featureFlags, 'getFeatureFlag').returns(false);
+    render(<App />);
+    assert.doesNotThrow(() => screen.getAllByTestId('legacy-app'));
```

Comment:
How do you feel about using `chai` `expect` in these new tests? That way we have a bit better error reporting and avoid needing `strictEquals` in many places.

Reply:
Ohh I missed that, yes. My intention was to keep consistent with the packages we use in other places, surely I missed this one. I will update that.